### ISSUE TITLE
Not able to generate model

### ISSUE THREAD

**Issue by sivagollapalli** [damln/localtower#13]
<img width="1271" alt="screen shot 2017-09-01 at 11 03 21 am" src="https://user-images.githubusercontent.com/280199/29956515-42a621aa-8f05-11e7-8673-89c7041316e7.png">

Parameters are 
```
{"utf8"=>"✓",
 "authenticity_token"=>"bxCtjP64fG5B2hdWJVgpo4sBPuZDfVhshbDqqb3mJpKwfBk9M0Up4GJblundWqlUkD04tro4xTq/kpLzyIVHcg==",
 "models"=>{"model_name"=>"something", "attributes"=>[{"attribute_name"=>"something", "attribute_type"=>"string", "defaults"=>"", "index"=>"true", "nullable"=>"true"}], "run_migrate"=>"false"}}
```

Version: 
Rails: `5.1.3`
Postgres: `9.6`

Note: I am using `localtower` with rails api

**Comment by damln** [comment]
Hi @sivagollapalli !
Thanks for the feedback I'm gonna have a look as soon as possible.
Are you using the latest version of Localtower? (0.3.1)

**Comment by sivagollapalli** [comment]
@damln Yes, I am using the latest version.

**Comment by damln** [comment]
Should be solved with version [0.4.1](https://rubygems.org/gems/localtower/versions/0.4.1)
